tips/TIP-0052/tip-0052.md

```diff
@@ -0,0 +1,345 @@
+---
+tip: 52
+title: Multi Address
+description: Defines an Address that is owned by multiple other addresses.
+author:
+  Eike Haß (@eike-hass ) <eike.hass@iota.org>, Max Hase (@muXxer) <maximilian.hase@iota.org>, Philipp Gackstatter
+  (@PhilippGackstatter) <philipp.gackstatter@iota.org>
+discussions-to: TODO
+status: Draft
+type: Standards
+layer: Core
+created: 2023-09-07
+---
+
+# Summary
+
+This TIP proposes a new Address type to allow multiple addresses to create complex multi-signature-like unlock
+conditions for outputs. The address is considered unlocked as soon as the cumulative weight of all unlocked contained
+addresses matches or exceeds the configured threshold, where each of the addresses has an assigned weight.
+
+# Motivation
+
+The current outputs in Stardust can be unlocked by single addresses, which does not allow outputs to be unlocked by
+multiple different entities, except with Ed25519 Threshold Signatures. The latter allows to model multi-control
+structures, but it is not transparent who the underlying controlling addresses are and in particular, they do not allow
+different Account or NFT Addresses to control an output. The _Multi Address_ introduced in this TIP allows for any other
+address type to control an output in a multi-control structure, in various configurations. The W3C DID Specification
+distinguishes between [independent control](https://www.w3.org/TR/did-core/#independent-control) and
+[group control](https://www.w3.org/TR/did-core/#group-control) which is a useful definition in this context. With the
+current approach only independent control can be modelled. Allowing for group control by multiple inspectable
+controlling entities might be useful in the following scenarios:
+
+- Modelling multiple W3C DID controllers.
+- Transparent modelling of ownership over funds or assets of entities like organizations.
+- Transparent modelling of committees for anchoring (e.g. L2)
+
+A _Multi Address_ allows for use-cases like:
+
+- Requiring a signature by a hardware security module (HSM) and a signature by one of two trusted devices through
+  software to unlock.
+- Requiring signatures of a supermajority of controlling parties each of which are an Account to unlock.
+- Modelling a 1 of n control structure in addition to a single backup address that has full control.
+- Allowing interested parties to inspect the conditions for unlocking, e.g. majority or supermajority.
+
+# Multi Address
+
+The following table shows the mapping from the address type of the **first byte** to the address type:
+
+| Address       | Type Byte as `uint8` | Bech32 Encoded |
+| ------------- | -------------------- | -------------- |
+| Multi Address | 40                   | iota1**q**...  |
```

Comment:
The example `iota19qq0ezu97zl76wqnpdxxleuf55gk0eqhscjtdgqm5sqwav6gcarz6vvesnk` given in this TIP doesn't adhere to this?

Reply:
Byte 40 is encoded to `9` in bech32, I might've confused that for a `q` :sweat_smile:. Updated.